### PR TITLE
fix bug when creating post without description

### PR DESCRIPTION
**Pull request by rostyk1331**

### CODE REVIEW

File: server/src/controllers/auth.controllers/auth.isAuthorized.ts [hunk 1]
```diff
@@ -5,16 +5,16 @@ export const isAuthorized = async (req: Request, res: Response, next: NextFuncti
...
         const user = await tokenVerification(token, res, next);
         if (!user) {
-            return next( {status: 401, message: 'Unauthorized'});
+            return next({status: 401, message: 'Unauthorized'});
         }
         res.locals.user = user;
         next();
     } catch (e) {
-        next( {status: 401, message: 'Unauthorized'});
+        next({status: 401, message: 'Unauthorized'});
```
Comment (InnaKutsik) [inline server/src/controllers/auth.controllers/auth.isAuthorized.ts]: DRY
{status: 401, message: 'Unauthorized'}
move to reuse constant

File: server/src/controllers/post.controllers/create.post.ts [hunk 2]
```diff
@@ -29,7 +29,8 @@ export const create = (req: Request, res: Response, next: NextFunction): void =>
             const description = req.body.description;
             const imgPath = req.file.location;
             const hashtagRegex = /\B(\#[a-zA-Z0-9]+\b)/g;
-            const tags = req.body.description.match(hashtagRegex);
+            const hashTags = description ? description.match(hashtagRegex) : [];
+            const tags = hashTags ? hashTags.map((tag: string) => tag.slice(1)) : [];
```
Comment (InnaKutsik) [inline server/src/controllers/post.controllers/create.post.ts]: description.match(hashtagRegex) is null if it is not match anything.
hashTags is array. Array is always true as array is object.

do you really need [] in ternary expression for hashTags  or maybe 
```const hashTags = description && description.match(hashtagRegex)``
then hashTags has more chances to be null.

File: server/src/controllers/auth.controllers/auth.isAuthorized.ts (hunk 1 above)
Comment (rostyk1331) [reply server/src/controllers/auth.controllers/auth.isAuthorized.ts]: Fixed.

File: server/src/controllers/post.controllers/create.post.ts (hunk 2 above)
Comment (rostyk1331) [reply server/src/controllers/post.controllers/create.post.ts]: I've been thinking about how to write this better, thanks. Fixed.

**Pull request merged**